src/main/java/core/model/HOModel.java

```diff
@@ -624,4 +626,9 @@ public void setYouthTrainings(List<YouthTraining> youthTrainings) {
                 .sorted(Comparator.comparing(YouthTraining::getMatchDate))
                 .collect(Collectors.toList());
     }
+
+    public void storeTrainingsSinceLastDownload() {
+        var recentTrainings = TrainingManager.instance().getRecentTrainings();
+        DBManager.instance().saveTrainings(recentTrainings,false);
```

Comment:
@akasolace i remember you had another approach to store recentTrainings. Feel free to replace this line by your approach. I use storeTrainingSinceLastDownload during download session just before subskill calculation is started.